airflow/providers/datadog/CHANGELOG.rst

```diff
@@ -19,6 +19,31 @@
 Changelog
 ---------
 
+2.0.2
+.....
+
+Features
+~~~~~~~~
+
+* ``Add documentation for January 2021 providers release (#21257)``
```

Comment:
is this one a mistake?
(I don't see this entry in other providers)

Reply:
Fixed!